samples/fluentbit/fluent-bit-ds.yaml

```diff
@@ -1,3 +1,16 @@
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: logging
+  labels:
+    name: logging
+---
+apiVersion: v1
```

Comment:
why do we need a new service account?

Reply:
The existing DaemonSet already required it, i just created it because the step was missing and it failed the deployment. Should it not use a new ServiceAccount?

Reply:
hm, not sure. ServiceAccount is used to grant access to resources in the cluster, does fluent bit need that?

Reply:
I'm gonna try removing it and see what happens.